src/main/java/mServer/crawler/sender/arte/ArteVideoDetailsDTO.java

```diff
@@ -0,0 +1,16 @@
+package mServer.crawler.sender.arte;
+
+public class ArteVideoDetailsDTO {
+    private String broadcastBegin = "";
+    
+    public ArteVideoDetailsDTO() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)